security_manager/app/it/gov/daf/securitymanager/service/RegistrationService.scala

```diff
@@ -21,7 +21,9 @@ object RegistrationService {
 
     if (user.userpassword.isEmpty || user.userpassword.get.length < 8)
       Future{Left("Password minimum length is 8 characters")}
-    else {
+    else if( !user.userpassword.get.matches("^[a-zA-Z0-9%@#   &,;:_'/\\\\<\\\\(\\\\[\\\\{\\\\\\\\\\\\^\\\\-\\\\=\\\\$\\\\!\\\\|\\\\]\\\\}\\\\)\u200C\u200B\\\\?\\\\*\\\\+\\\\.\\\\>]*$") )
```

Comment:
... why?

Reply:
CKAN login doesn't accept password with several special chars. It seems this happens with all localized type chars (é°à£òçÜüä...)